src/proof_system/widget/ecc/scalar_mul/fixed_base/proverkey.rs

```diff
@@ -151,3 +150,17 @@ impl ProverKey {
         q_fixed_group_add_poly * &(a * ecc_separation_challenge)
     }
 }
+
+pub(crate) fn extract_bit(
+    curr_acc: &BlsScalar,
+    next_acc: &BlsScalar,
+) -> BlsScalar {
+    // Next - 2 * current
+    next_acc - (curr_acc + curr_acc)
```

Comment:
You can save two character and having`next_acc - curr_acc - curr_acc`, and it's probably clearer that you're subtracting `curr_acc` twice from `next_acc`.